easybuild/easyblocks/s/suitesparse.py

```diff
@@ -161,8 +161,13 @@ def install_step(self):
 
     def make_module_req_guess(self):
         """Add config dir to CPATH so include file is found."""
+        """Add UMFPACK adn AMD library dirs needed for scikit-umfpack."""
```

Comment:
double docstring (and typo), fixed with https://github.com/hpcuantwerpen/easybuild-easyblocks/pull/1

@sbecuwe: please merge?